.circleci/config.yml

```diff
@@ -55,20 +55,90 @@ jobs:
             pip install --upgrade pip setuptools
             pip install -r requirements.txt
             pip install -r deid/requirements.txt
+            pip install -r dev_requirements.txt
       - run:
           name: Show bashrc file
           command: cat ${BASH_ENV}
       # Test
       #   This would typically be a build job when using workflows, possibly combined with build
       - run:
           name: Run unit tests
+          working_directory: ~/curation
+          command: |
+            mkdir -p tests/results/coverage/unit/xml
+            mkdir -p tests/results/coverage/unit/html
+            mkdir -p tests/results/junit/unit
```

Comment:
What do the curation tests have to do with junit? Am I missing something? Combine step doesn't seem to touch this

Reply:
You are missing the difference in coverage reporting and CircleCI Test Summary reporting.  The CircleCI platform requires junit xml reporting to use the Test Summary feature.  I like having the test summary data consolidated into the single tab at the beginning of the build, especially when I'm looking for the failures.  

No, the combine step doesn't touch the junit files because it is creating a combined coverage file to show us which lines of our source code were tested and which were not.  I run tests as unit and integration separately, so I have to combine them to get the full report.  Coverage does not use the junit results.  It uses the coverage results.

Reply:
Done.